app/store.go

```diff
@@ -0,0 +1,146 @@
+package app
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"time"
+
+	"github.com/datagovsg/nomad-parametric-autoscaler/logging"
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+// Store records new state updates to a persistent store
+// table schema : timestamp (TIMESTAMP) | state (text)
+type Store struct {
+	db *sqlx.DB
+}
+
+// CreateTablesSQL contains statement for table if doesnt exist
+const CreateTablesSQL = `
+CREATE TABLE autoscaler (
+    timestamp TIMESTAMP,
+    state text
+);
```

Comment:
Can use `createTablesSQL` instead, since it's need not be a public value (it's only used additionally in test in the same module). I'm surprised `go fmt` didn't comment about this though.